src/readbin/symbols.mli

```diff
@@ -26,3 +26,9 @@ open Bap.Std
 *)
 
 val read : ?demangle:Options.demangle -> filename:string -> arch -> mem -> string table
+
+val read_addrset : string -> Addr.Hash_set.t
+
+val write_addrset : ?filename:string -> Addr.Hash_set.t -> unit
```

Comment:
a better design choice would be to pass `out_channel` instead of optional filename. But for consistence you need to fix other functions io functions in this module as well.